frontend/src/UserModule/Login/components/LoginPage.vue

```diff
@@ -19,9 +19,16 @@ export default {
           password: this.password,
         })
         .then((response) => {
-          // Save the access token in local storage or Vuex store
+          // Save the access token in cookie
           const accessToken = response.data.access;
-          localStorage.setItem('accessToken', accessToken);
+          const refreshToken = response.data.refresh;
+
+          // save another cookie, expiration, set to 5 minutes so we can
+          // refresh the token
+          const expires = new Date(Date.now() + 300 * 1000);
+          this.$cookies.set('token', accessToken);
+          this.$cookies.set('refresh', refreshToken);
```

Comment:
Whats this for?

Reply:
Refresh token is provided to /auth/token/refresh/ to obtain a new access token. Refresh token expires 1 day, access token 5 min

Reply:
But it doesn't look like you have set an expiration in cookie for the refresh token?, can you also update the key name to refresh-token?